test-suite/src/test/scala/org/scalajs/testsuite/javalib/CollectionTest.scala

```diff
@@ -165,7 +165,7 @@ trait CollectionTest extends JasmineTest with ExpectExceptions {
       coll.add("one")
       expect(coll.contains("one")).toBeTruthy
       expect(coll.contains("two")).toBeFalsy
-      expect(coll.contains(null)).toBeFalsy
+      //expect(coll.contains(null)).toBeFalsy
```

Comment:
I would rather add a new condition `def allowsNullElementQuery: Boolean` because until now the ported collections that did not support the insertion of `null` elements did return `false` on `.contains(null)`. We should not stop testing and hence a new condition is required.